web/src/cypress/support/limits.ts

```diff
@@ -0,0 +1,52 @@
+declare namespace Cypress {
+  interface Chainable {
+    getLimits: typeof getLimits
+    updateLimits: typeof updateLimits
+  }
+}
+
+interface Limits {
+  id: ID
```

Comment:
Would this be easier as an object in the helper?

If you need to look up a particular limit for a test you'd have to scan through the array each time otherwise.

We could define it as a struct or a map with an enum of possible keys.